Makefile

```diff
@@ -216,12 +218,16 @@ endif
 		--build-arg NVIDIA=$(NVIDIA) --build-arg ov_contrib_branch="$(OV_CONTRIB_BRANCH)" \
 		-t $(OVMS_CPP_DOCKER_IMAGE)-build:$(OVMS_CPP_IMAGE_TAG)$(IMAGE_TAG_SUFFIX) \
 		--build-arg JOBS=$(JOBS)
+
+targz_package:
```

Comment:
`make targz_package` will fail due to missing prerequisite (ovms_build). Should we handle it somehow? Require the step? `targz_package: ovms_build`

Reply:
Ok.